datafeeder/src/main/java/org/georchestra/datafeeder/autoconf/PostgisSchemasConfiguration.java

```diff
@@ -0,0 +1,134 @@
+package org.georchestra.datafeeder.autoconf;
+
+import com.google.common.annotations.VisibleForTesting;
+import lombok.*;
+import org.springframework.beans.factory.InitializingBean;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.util.Assert;
+import org.springframework.util.StringUtils;
+import org.springframework.validation.annotation.Validated;
+
+import java.time.Duration;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+@Data
+@EqualsAndHashCode(exclude = "includesBySchema")
+@ConfigurationProperties(prefix = "postgis.schemas")
+public class PostgisSchemasConfiguration implements InitializingBean {
```

Comment:
Is it copy/pasted from the data-api codebase (https://github.com/georchestra/data-api/blob/main/src/services/ogc-features/src/main/java/com/camptocamp/opendata/ogc/features/autoconfigure/postgis/PostgisSchemasConfiguration.java) ? Then maybe we should add a comment to point the original file at least.